src/mmrelay/matrix_utils.py

```
@@ -4,6 +4,95 @@
+    "on_room_member",
+    "on_room_message",
+    "queue_message",
+    "save_credentials",
+    "send_image",
+    "send_reply_to_meshtastic",
+    "send_room_image",
+    "send_text_reply",
+    "strip_quoted_lines",
+    "time",
+    "upload_image",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)